test/e2e/upg_e2e.go

```diff
@@ -534,6 +534,47 @@ var _ = ginkgo.Describe("Clearing message queue", func() {
 	})
 })
 
+var _ = ginkgo.Describe("Quota Validation Tests", func() {
+	ginkgo.Context("Quota Validity Time", func() {
+		f := framework.NewDefaultFramework(framework.UPGModeTDF, framework.UPGIPModeV4)
+		ginkgo.It("should generate usage report", func () {
+			ginkgo.By("Creating session with URR")
+			sessionCfg := &framework.SessionConfig{
+				IdBase: 1,
+				UEIP:   f.UEIP(),
+				Mode:   f.Mode,
+			}
+			reportCh := f.PFCP.AcquireReportCh()
+			triggers := 0x80
+			framework.Logf("Trigger %d", triggers)
```

Comment:
`0x%04x` would perhaps be more readable in the output than `%d`? (if this is really needed at all)

Reply:
forgot to remove :)